content/id/docs/tasks/administer-cluster/configure-upgrade-etcd.md

```diff
@@ -0,0 +1,234 @@
+
+   1. Perbarui flag `--etcd-server` untuk membuat Kubernetes mengetahui perubahan konfigurasi, lalu mulai ulang server API Kubernetes.
+   2. Perbarui konfigurasi load balancer jika load balancer digunakan dalam penerapan.
+
+Untuk informasi lebih lanjut tentang konfigurasi ulang klaster, lihat [Dokumentasi Konfigurasi etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member).
+
+## Mencadangkan klaster etcd
+
+Semua objek Kubernetes disimpan di etcd. Mencadangkan secara berkala data klaster etcd penting untuk memulihkan klaster Kubernetes di bawah skenario bencana, seperti kehilangan semua node master. File _snapshot_ berisi semua status Kubernetes dan informasi penting. Untuk menjaga data Kubernetes yang sensitif aman, enkripsi file snapshot.
+
+Mencadangkan klaster etcd dapat dilakukan dengan dua cara: _snapshot_ etcd bawaan dan snapshot volume.
+
+### Snapshot bawaan
+
+etcd mendukung _snapshot_ bawaan, jadi mencadangkan klaster etcd lebih mudah. _snapshot_ dapat diambil dari anggota langsung dengan command `etcdctl snapshot save` atau dengan menyalin `member/snap/db` file dari etcd [direktori data](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/configuration.md#--data-dir) yang saat ini tidak digunakan oleh proses etcd. Mengambil snapshot biasanya tidak akan memengaruhi kinerja anggota.
+
+Di bawah ini adalah contoh untuk mengambil snapshot dari _keyspace_ yang dilayani oleh `$ENDPOINT` ke file `snapshotdb`:
+
+```sh
+ETCDCTL_API=3 etcdctl --endpoints $ENDPOINT snapshot save snapshotdb
+# exit 0
+
+# verify the snapshot
+ETCDCTL_API=3 etcdctl --write-out=table snapshot status snapshotdb
++----------+----------+------------+------------+
+|   HASH   | REVISION | TOTAL KEYS | TOTAL SIZE |
++----------+----------+------------+------------+
+| fe01cf57 |       10 |          7 | 2.1 MB     |
++----------+----------+------------+------------+
+```
+
+### Snapshot volume
+
+Jika etcd berjalan pada volume penyimpanan yang mendukung cadangan, seperti Amazon Elastic Block Store, buat cadangan data etcd dengan mengambil snapshot dari volume penyimpanan.
+
+## Scaling up klaster etcd
+
+Penskalaan klaster etcd meningkatkan ketersediaan dengan menukar kinerja. Penskalaan tidak meningkatkan kinerja atau kemampuan klaster. Aturan umum adalah untuk tidak _scale up_ atau _scale down_ klaster etcd. Jangan mengkonfigurasi grup penskalaan otomatis untuk klaster etcd. Sangat disarankan untuk selalu menjalankan klaster etcd five-member statis untuk klaster Kubernetes produksi pada skala yang didukung secara resmi.
+
+Penskalaan yang wajar adalah untuk meningkatkan klaster three-member menjadi five-member, ketika dibutuhkan lebih banyak reliabilitas. Lihat [Dokumentasi Rekonfigurasi etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member) untuk informasi tentang cara menambahkan anggota ke klaster yang ada.
```

Comment:
"Penskalaan" is not formal, use "penyekalaan". https://kbbi.kemdikbud.go.id/entri/penyekalaan
```suggestion
Penyekalaan yang wajar adalah untuk meningkatkan klaster dengan tiga anggota menjadi dengan lima anggota, ketika dibutuhkan lebih banyak reliabilitas. Lihat [Dokumentasi Rekonfigurasi etcd](https://github.com/coreos/etcd/blob/master/Documentation/op-guide/runtime-configuration.md#remove-a-member) untuk informasi tentang cara menambahkan anggota ke klaster yang ada.
```